src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -222,6 +220,26 @@ private LogQuery buildQuery(String offset, AdminActivityLogPageData data) {
         }
         
         String status="&nbsp;&nbsp;Total Logs gone through in last search: " + totalLogsSearched + "<br>";
+        
+        if (earliestLogChecked != null) {
+            String userGoogleId = earliestLogChecked.getId();
+            String userRole = earliestLogChecked.getRole();
+            
+            SimpleDateFormat sdf = new SimpleDateFormat("dd-MM-yy HH:mm:ss");
+            sdf.setTimeZone(TimeZone.getTimeZone(Const.SystemParams.ADMIN_TIME_ZONE));
+            
+            TimeZone adminTimeZone = TimeZone.getTimeZone(Const.SystemParams.ADMIN_TIME_ZONE);
+            Calendar adminTimezoneCalendar = Calendar.getInstance(adminTimeZone);
+            adminTimezoneCalendar.setTimeInMillis(earliestLogChecked.getTime());
+            
+            String timeInAdminTimeZone = sdf.format(adminTimezoneCalendar.getTime());
+            String timeInUserTimeZone = getLocalTimeInfo(userGoogleId, userRole, String.valueOf(earliestLogChecked.getTime()));
+            
+            status += "The earliest log entry checked on <b>" + timeInAdminTimeZone + "</b> in Admin Time Zone and on <b>" + timeInUserTimeZone + "</b> in Local Time Zone.<br>";
```

Comment:
Here you are assuming that: earliest search time = log time of the earliest log
This may not be always true.

We have to consider the case where we search from lets say 10/10/2015 10:00 to 11/10/2015 10:00 and there are logs only from 11/10/2015 02:00. Currently, the earliest time will show as 11/10/2015 02:00 rather than 10/10/2015 10:00